src/controllers/__tests__/products.test.js

```diff
@@ -70,4 +73,75 @@ describe('Products routes', () => {
       expect(res.body.data).toEqual(expect.objectContaining(mProduct));
     });
   });
+  describe('PATCH /:id', () => {
+    beforeEach(async () => {
+      // create user for authentication
+      const user = {
+        firstName: 'Glenn',
+        lastName: 'Quagmire',
+        email: 'glennQQQ@email.com',
+        schoolName: 'Yale University',
+        password: 'gleN123',
+      };
+      await server.post('/api/auth/signup').send(user);
+      // create product
+      const product1 = {
+        title: 'Javascript Book',
+        description: 'Book to learn english',
+        price: '80',
+        category: 'Book',
+        coverImage: 'www.book.jpg',
+        type: 'Product',
+        location: 'Ankara',
+        condition: 'Used',
+      };
+      const product2 = {
+        title: 'Javascript Book2',
+        description: 'Book to learn english2',
+        price: '80',
+        category: 'Book',
+        coverImage: 'www.book.jpg',
+        type: 'Product',
+        location: 'Ankara',
+        condition: 'Used',
+      };
+      myproduct1 = await Product.create(product1);
+      myproduct2 = await Product.create(product2);
+    });
+
+    test('Update given field, return with 200 status code', async () => {
+      const toUpdate = { title: 'Potato' };
```

Comment:
It's better to test all the updatable fields. so along `title`, put all the fields that could be updated, and of course, check them below, if they were updated (expect)